types/jsoneditor-for-react/index.d.ts

```diff
@@ -10,6 +10,7 @@ import JSONEditor, { JSONEditorOptions } from 'jsoneditor';
 
 export interface ReactJsonEditorProps {
     values: {};
+    onChange: (newValues: {}) => void;
```

Comment:
While `{}` could definitely be the type of the value, it might not work for everyone as there are more valid json types. In the types of `jsoneditor` it's typed as `any`, but we would also use a safer (recursive) type like:

`type Json = null | boolean | number | string | Json[] | { [prop: string]: Json }`

[Source](https://github.com/microsoft/TypeScript/issues/1897#issuecomment-822032151)

Reply:
Thank you for the feedback. I will make the changes and submit again.